can/interfaces/vector/canlib.py

```diff
@@ -10,22 +10,16 @@
 import logging
 import time
 import os
-from typing import List, NamedTuple, Optional, Tuple, Sequence, Union
+from types import ModuleType
+from typing import List, NamedTuple, Optional, Tuple, Sequence, Union, Any, Dict
 
 try:
     # Try builtin Python 3 Windows API
     from _winapi import WaitForSingleObject, INFINITE
 
     HAS_EVENTS = True
 except ImportError:
-    try:
-        # Try pywin32 package
-        from win32event import WaitForSingleObject, INFINITE
-
-        HAS_EVENTS = True
-    except ImportError:
-        # Use polling instead
-        HAS_EVENTS = False
```

Comment:
I'm unsure about this change, should we really abandon this fallback? The module seems to be private, are we sure it will stay around?

If yes, we should remove this from the docs:

> If you are using Python 2.7 it is recommended to install pywin32, otherwise a slow and CPU intensive polling will be used when waiting for new messages.

`pywin32` is still specified in the `setup.py`, but I suppose it's used elsewhere.

Reply:
`_winapi` is only available in CPython. pywin32 is also only available for CPython. So unless CPython removes `_winapi` (why would they?) that code is pointless. I can add a test for that unlikely case :smile:

pywin32 is still used in BroadcastManager.

Reply:
I added the test and apparently PyPy supports `_winapi` too.

Reply:
🥳 Then maybe let that test run there too? Else, nice work!

Reply:
The test runs on PyPy too.

Reply:
Ah, sorry, of course!